deepsensor/model/model.py

```diff
@@ -976,13 +1068,16 @@ def stitch_clipped_predictions(
             # Append patchwise DeepSensor prediction object to list
             preds.append(pred)
 
-        
-        overlap_norm = tuple(patch - stride for patch, stride in zip(patch_size, stride))
+        overlap_norm = tuple(
+            patch - stride for patch, stride in zip(patch_size, stride)
+        )
         patch_overlap_unnorm = get_patch_overlap(overlap_norm, data_processor, X_t)
```

Comment:
Still missing the `x1_ascending` and `x2_asencding` args here, but I know you're resolving elsewhere.